src/views/FindOrder.vue

```diff
@@ -104,61 +104,80 @@
-                      <p>{{ $t("Color") }} : color</p>
-                      <p>{{ $t("Size") }}: size</p>
+                      <p> {{ getProduct(item.productId).brandName ? getProduct(item.productId).brandName : '-' }} </p>
+                      {{ item.virtualProductName }}
+                      <p> {{ $t("Color") }} : {{ $filters.getFeature(getProduct(item.productId).featureHierarchy, '1/COLOR/') }} </p>
+                      <p> {{ $t("Size") }} : {{ $filters.getFeature(getProduct(item.productId).featureHierarchy, '1/SIZE/') }} </p>
                     </ion-label>
-                    <ion-badge color="primary" slot="end">Approved</ion-badge>
-                  </ion-item>
-                  <ion-item>
-                    <ion-label> {{ $t("Promise date") }} </ion-label>
-                    <p slot="end">14 Jan 2021</p>
-                  </ion-item>
-                  <ion-item>
-                    <ion-label> {{ $t("PO arrival date") }} </ion-label>
-                    <p slot="end">14 Jan 2021</p>
-                  </ion-item>
-                  <ion-item>
-                    <ion-label> {{ $t("Last brokered") }} </ion-label>
-                    <p slot="end">California Warehouse</p>
+                    <ion-badge color="primary" slot="end"> {{ item.orderItemStatusId }} </ion-badge>
                   </ion-item>
+                  <!-- TODO: Need to handle this property -->
+                  <div v-if="item.preOrderStatus || item.backOrderStatus || item.unFillable">
+                    <ion-item>
+                      <ion-label> {{ $t("Promise date") }} </ion-label>
+                      <p slot="end"> {{ item.promisedDatetime ? $filters.formatUtcDate(item.promisedDatetime, 'YYYY-MM-DDTHH:mm:ssZ') : '-'  }} </p>
+                    </ion-item>
+                    <ion-item>
+                      <ion-label> {{ $t("PO arrival date") }} </ion-label>
+                      <!-- TODO: Need to handle this property -->
+                      <p slot="end"> {{ item.promiseOrderArrivalDate ? item.promiseOrderArrivalDate : '-' }} </p>
+                    </ion-item>
+                    <ion-item>
+                      <ion-label> {{ $t("Last brokered") }} </ion-label>
+                      <!-- TODO: Need to handle this property -->
+                      <p slot="end"> {{ item.lastBrokered ? item.lastBrokered : '-' }} </p>
+                    </ion-item>
+                  </div>
+                  <div v-else>
+                    <ion-item>
+                      <ion-label> {{ $t("Shipping method") }} </ion-label>
+                      <p slot="end"> {{ item.shipmentMethodTypeId }} </p>
+                    </ion-item>
+                    <ion-item>
+                      <ion-label> {{ $t("Shipping from") }} </ion-label>
+                      <p slot="end"> {{ $filters.getShippingFrom(item.orderRoles, '1/Ship-From Vendor/') ? $filters.getShippingFrom(item.orderRoles, '1/Ship-From Vendor/') : "-" }} </p>
```

Comment:
we should probably be using orderFacilityName or [facilityName](https://git.hotwax.co/commerce/oms/blob/develop/applications/hwmapps/solrconfig/solrschema.xml#L198)

@dixitdeepak @patelanil what do you think?